functions.yml

```diff
@@ -153,17 +163,21 @@ etl:
           [Row(calls=[-1, -1])]
       args:
         - name: probabilities
-          doc: Probabilities
+          doc: The array of probabilities to convert
         - name: numAlts
-          doc: The number of alts
+          doc: The number of alternate alleles
         - name: phased
-          doc: Whether the probabilities are phased or not
+          doc: Whether the probabilities are phased. If phased, we expect one 2 * numAlts values 
+            in probabilities array. If unphased, we expect one probability per possible genotype.
```

Comment:
`in probabilities array` -> `in the probabilities array`

Reply:
Done